custom-vu/src/main/kotlin/jces1209/vu/page/boards/view/cloud/NextGenBoardPage.kt

```diff
@@ -32,6 +36,11 @@ class NextGenBoardPage(
         return NextGenBoardContent(issueCards)
     }
 
+    override fun previewIssue(): BoardPage {
+        logger.debug("Issue bento view is not implemented")
```

Comment:
This is interesting. So as a user, on a next-gen board, I see an issue card and I want to see the issue details. Can I click on the card? What would happen - is it a sidebar issue view or is it a popup? Or nothing? Do I have to copy-paste the issue key and go via URL?

One way or another, as a user I will get into the issue details. Our task is to represent the most popular way. And measure it.

Because currently, if we just skip this part of the user journey, the next-gen boards get a very fast measurement, which doesn't represent what the user feels.